### ISSUE TITLE
what about italic and condensed

### ISSUE THREAD

**Issue by alexmyczko** [theleagueof/league-spartan#7]
would they be hard to add?

**Comment by dmoles** [comment]
There's one take on an italic [here](https://github.com/MircoMonsees/League-Spartan-Italic).

**Comment by cssobral2013** [comment]
Regarding Italics, I'm proposing them.

**Comment by alerque** [comment]
Hey @MircoMonsees I see you worked on an Italic style at some point a couple years ago. After many years of inactivity we finally have some motion in getting League font projects updated now. This family has been overhauled completely. I was wondering if you might be interested in helping out in contributing your italic style and getting it setup for integration into this project and to be distributed with the official releases?

**Comment by MircoMonsees** [comment]
Hey all, I designed the italic in conjunction with Tyler Fincks weight expansion and variable font conversion a few years ago. It is basically done (barring some testing) but was never released. I’d be up for integrating it into this project but I will need some guidance on how to make it happen. Thanks, Mirco

**Comment by alerque** [comment]
I had a look at your repository — and lets just say the experience made me run to chocolate for comfort.

> ![XKCD 1459](https://imgs.xkcd.com/comics/documents_2x.png)

No personal offense meant of course, but at least a couple years ago when you did that work I think your Git foo was kind of weak. Perhaps the best thing to do to integrate the Italic sources into this project would be for me to clone the repo, do some history rewrites to remove the unrelated bits and unravel the *actual* progression of versions, then splice just the sources that survive that process into this repo. How does that sound?

If it's a plan, maybe you can spend a couple minutes poking through the file list from each [commit in the history], then spend a couple more minutes giving me a rough summary of the "progression". I'm particularly confused about why there are older commits and files with file names that have higher version numbers (6, 7, 8, 9, etc.) than later ones (the final commit settles on 5). Don't bother trying to fix it, just reconstruct with a brief description what the development process was as far as how files were shuffled around (shared with Tyler?) and I'll try to work from there to reconstruct a linear commit history with just the Italic sources.

**Comment by MircoMonsees** [comment]
I agree that the repo seems a bit obtuse and scattered (even for myself, coming back to it 2 years later). This is mostly because I used it as more of an open-source dropbox where I cloned my personal working folder for anyone interested. Commits were mostly just me filling gaps or moving files around, they don't always correspond to actual changes.

The project started when I approached Tyler about doing the italic. He was about 90% through the process of expanding and variable-izing the roman at the time. I just pulled a copy of the glyphs file and worked on the italic standalone, checking back with him occasionally and copying changes from the roman by hand When it comes to the italic itself, I am the only person who ever touched the working files. 

Here's a quick version rundown:
v1 to v16 - incomplete, bad drawing quality, trying out different ideas etc. (I uploaded most of these all at once when I created the repo)
R1 - R4 - Release candidates 1 to 4, these versions were overall good and had no glaring issues (seeing as the numbers have gone up, there were still things I wasn't happy with)
R5 - Current version (Aug 2018 - identical in features to Tyler's roman at that time)

**Comment by alerque** [comment]
Great, that helps a lot @MircoMonsees. When I get a chance I'll see what can be done about grafting the relevant bits of that history into this repo as a PR. I'll ping you if there are any questions and certainly to look it over if I get it working.

**Comment by alerque** [comment]
Oh oops I almost forgot about the other half of this mess @MircoMonsees ... there seems to have been two parallel efforts at italics, yours being just one of them. I was wondering if you ever reviewed the fonts in #22 and whether you have any comment as to their quality or the difference in designs with your own.

**Comment by MircoMonsees** [comment]
I had a quick look (not exhaustive so forgive me if I missed something) and it looks like that italic designs are just slanted/oblique versions of the roman. My italic is a "real" italic with adjusted strokes and skeletons. 

![30080043_243347009573740_8650671077447434240_n](https://user-images.githubusercontent.com/15860083/92977052-df272880-f483-11ea-99eb-e2f318816133.jpg)

**Comment by alerque** [comment]
Thanks, that's kind of what I suspected but I hadn't gotten a chance to actually put them side by side yet, or even look at them really.

**Comment by MircoMonsees** [comment]
I realized that there is more work involved in integrating the italic with the roman then I initially thought.
My current design has no extralight or black weight; the current master weights correspond roughly to light and extra bold of the roman. To fix this, the following has to be done:
1) New designs (lowercase glyphs plus some others)
Redesigning the stroke weight of my light master to match extralight and for extrabold to match black
2) Adapted Glyphs (Uppercase, punctuation, numbers etc)
Copying over the designs from the roman and making them compatible.
3) Testing and interpolation

Its a decent amount of work but I can do it. Not sure what kind of timeline to put on this though.

**Comment by alerque** [comment]
Roger that.

I think the next step is still for me to get the initial splice of sources going in a branch, then leave you to finish up the adaptations that need to happen.

**Comment by iainhallam** [comment]
@alerque @MircoMonsees did you ever get anywhere with creating a branch for this? A true italic in League Spartan would be great.
